src/frontend/js/types/Joanie.ts

```diff
@@ -170,12 +170,22 @@ export interface Enrollment {
   products: Product[];
 }
 
+// Contract
+
+export interface Contract {
+  contract_id: string;
+  learner_name: string;
+  product_title: string;
+  nb_signatures: number;
+}
+
 // Order
 export enum OrderState {
-  CANCELED = 'canceled',
-  FAILED = 'failed',
```

Comment:
This state [do not exist on joanie](https://github.com/openfun/joanie/blob/main/src/backend/joanie/core/enums.py#L53-L57)

Reply:
Order states have changed after this issue and related PR: https://github.com/openfun/joanie/issues/363

Reply:
I think i've now the right states between front and back.